qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/project/MockQuteProject.java

```diff
@@ -161,4 +162,18 @@ protected static ValueResolverInfo createValueResolver(String namespace, String
 
 	protected abstract Map<String, NamespaceResolverInfo> createNamespaceResolverInfos();
 
+	public JavaMethodInfo getMethodValueResolver(String typeName, String methodName) {
+		try {
+			List<MethodValueResolver>  resolvers = super.getDataModelProject().get().getMethodValueResolvers();
+			for (MethodValueResolver resolver : resolvers) {
+				if (typeName.equals(resolver.getSourceType()) && methodName.equals(resolver.getMethodName())) {
+					return resolver;
+				}
+			}
+		} catch (Exception e) {

```

Comment:
would it be useful to at least log the error?

Reply:
I can do that, but it is in the test and in this case CompletableFuture is resolved immediatly. Is it really interesting to log that?

Reply:
that's why I'm asking. if not useful then ignore it.